### PR TITLE
Otto theme

### PR DESCRIPTION
**Pull request by ignaciocastro**
Hoping everything's ok.

### CODE REVIEW

File: Otto/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Otto
+
+## Screenshots
+
+![Otto1](https://raw.githubusercontent.com/ignaciocastro/spicetify-themes/master/Otto/Otto.png)
+![Otto2](https://raw.githubusercontent.com/ignaciocastro/spicetify-themes/master/Otto/Otto2.png)
```
Comment (morpheusthewhite) [inline Otto/readme.md:6]: Please use relative paths (look at other readmes which refer to pictures in the same folder)

**Comment by ignaciocastro** [comment]
Fixed.

File: Otto/readme.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 ## Screenshots
```
Comment (morpheusthewhite) [inline Otto/readme.md:4]: Why not adding also the other screenshots?

Comment (morpheusthewhite) [reply Otto/readme.md:4]: I mean both `Otto.png` and `Otto2.png`

**Comment by ignaciocastro** [comment]
I fixed the colors and updated the screenshots, but forgot to delete the older, unused ones. Fixed!

**Pull request merged**